docs/_includes/common/success-message.html

```diff
@@ -0,0 +1,65 @@
+<div id="success-message">
+    <h2><span class="secondary-text">@ebay/skin/</span>success-message</h2>
+
+    <p>A success-message is a message of celebration used when a user has completed something important. Its higher energy than a confirmation notice and is always the major focus of the page.</p>
+    <p>It can be used within a dialog, or as a full screen in mobile applications.</p>
+
+    <h3>With an action button</h3>
+
+    <div class="demo">
+        <div class="demo__inner" style="width: 615px; max-width:90vw">
```

Comment:
Why the width constraint?

Reply:
Yeah, this isn't the only place we have fixed widths in demos. As it sometimes more accurately reflects the use case.

Reply:
Actually though, looking at the demos now and I'm not sure they all do need these constraints in the examples.

Reply:
I think the first two demos look fine at full width. Maybe just me though.

Reply:
Yeah, it just wraps the content.  It more accurately shows what the design playbook asks for, but its not needed there really.  Only the `full` one that fills a dialog is needed.